atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/ClojureObjectUtils.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.jepsen;
+
+import clojure.lang.Keyword;
+import clojure.lang.PersistentArrayMap;
+
+public final class ClojureObjectUtils {
+    private ClojureObjectUtils() {
+    }
+
+    public static boolean mapContainsKeywordNamed(PersistentArrayMap map, String keyName) {
+        for (Object keyObject : map.keySet()) {
+            if (isKeywordNamed(keyName, keyObject)) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
Alternatively, 

```java
Set<?> mapKeys = map.keySet();
return mapKeys.stream().anyMatch(keyObject -> isKeywordNamed(keyName, keyObject));
```

(or you could inline `mapKeys` even, though that generates warnings about raw types - though because we are actually using `Object` it's okay here)

Feel free to push back on this one, I probably like functional stuff a bit too much for my own good.